robot.py

```diff
@@ -14,13 +14,17 @@ def teleopPeriodic(self):
         self.robot_drive.arcadeDrive(self.stick)
         print(self.l_motor.get())
 
+        drive_right_y = self.driver.getY(RIGHT)
```

Comment:
Hm, these changes look like they snuck in here from another branch. Ideally we wouldn't have these, but it won't cause any trouble to leave them in this time. Try to have cleaner commits in the future.